index.html

```diff
@@ -1,50 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="shortcut icon" href="images/YT.png" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>YouTube Website by Yayner</title>
-  </head>
-  <body>
-    <nav class="nav-flex">
-      <div class="nav-left nav-flex">
-        <img src="images/menu.png" class="menu" />
-        <img src="images/logo.png" class="logo" />
-      </div>
-      <div class="nav-middle nav-flex">
-        <div class="search-div nav-flex">
-          <input type="text" placeholder="Search" />
-          <img src="images/search.png"/>
-        </div>
-        <img src="images/voice-search.png" class="voice-search" />
-      </div>
-      <div class="nav-right nav-flex">
-        <img src="images/upload.png" />
-        <img src="images/more.png" />
-        <img src="images/notification.png" />
-        <img src="images/profile.jpeg" class="user-pic" />
-      </div>
-    </nav>
-    <!-- ------------sidebar-------------- -->
-    <div class="sidebar">
-      <div class="sidebar-links">
-        <!-- content dynamically rendered, requires javascript enabled -->
-      </div>
-      <div class="subscribed-list">
-        <h3>SUBSCRIPTIONS</h3>
-        <!-- content dynamically rendered, requires javascript enabled -->
-      </div>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="shortcut icon" href="images/YT.png" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>YouTube Website by Yayner</title>
+</head>
+
+<body>
+  <nav class="nav-flex">
+    <div class="nav-left nav-flex">
+      <img src="images/menu.png" class="menu" />
+      <img src="images/logo.png" class="logo" />
     </div>
-    <!-- ------------------ main----------------------- -->
-    <div class="container">
-      <div class="video-container">
-        <!-- content dynamically rendered, requires javascript enabled -->
+    <div class="nav-middle nav-flex">
+      <div class="search-div nav-flex">
+        <input type="text" placeholder="Search" />
+        <img src="images/search.png" />
       </div>
+      <img src="images/voice-search.png" class="voice-search" />
```

Comment:
Please give an alt attribute to your img tag. This will enable screen readers easily grasp what Is being written if the image doesn't show up